### ISSUE TITLE
Epic: convert KubeStellar to use edgeDeployment bundle

### ISSUE THREAD

**Issue by clubanderson** [kubestellar/kubestellar#1128]
### Feature Description

it would be easier to bundle resources, internal to KubeStellar, to reduce the need for multiple workspaces to pass individual yaml definitions around.  This will make it easier to marshall and apply what is dropped in the WDS.  This also will help with automatic creation of edgePlacement objects that contain an enumeration of namespaces objects.  Additionally, the automation can update the edgePlacement objects to include clusterscoped object references as feedback from the KubeStellar syncer attempts to apply namespaced objects at the target.

This approach also allows the removal of, at least, 1 of the 4 workspaces in KubeStellar.  Mailbox space can be removed.  Mailbox namespace can be removed because all objects needed for the KubeStellar syncer can be stored as edgeDeployment objects in the KubeStellar namespace.  Removing this namespace and updating the controllers, installation, and kubectl plugins to interact with namespaces instead of spaces reduces cognitive load on our users and removes confusing instructions and reduces length of documentation.

KubeStellar Syncer will be updated to watch the KubeStellar namespace for objects destined to it's specific target
Placement Translator will be updated to create edgeDeployment bundles in the KubeStellar namespace
Where Resolver will be updated to combine automatically created edgePlacements in the WDS with location objects in the KubeStellar namespace
SyncerConfig objects will now be created in the KubeStellar namespace

The result is a true workload native experience for our customers

### Proposed Solution
- [ ] create a simple edgeDeployment CRD that contains the yaml for the namespaced objects to be applied. #1129 
- [x] Update Placement Translator to concatenate individual yaml into edgeDeployment objects in KubeStellar namespace #1130 
- [x] Update Mailbox Controller to create syncerconfigs in KubeStellar namespace #1131 
- [x] Remove MB space #1132 
- [ ] Update KubeStellar Syncer to fetch and apply edgeDeployment from KubeStellar namespace #1135 
- [ ] Change name of WDS to PostOfficeSpace to help with user experience #1136 

### The following items are on hold
- [x] Remove IS space - locations and synctargets move to KubeStellar namespace #1133 
- [x] Update Where Resolver to combine locations and synctargets from KubeStellar namespace #1134 

### Want to contribute?

- [ ] I would like to work on this issue.

**Comment by nirrozenbaum** [comment]
this conversion is part of the pluggable transport implementation

**Comment by nirrozenbaum** [comment]
as mentioned in previous comment this is part of pluggable-transport.
as the team decided to start with OCM plugin in phase1 and get back to KS built-in syncer only in phase2 - I think this can be closed for now as non-relevant.
we will complete the transition to ks-0.20, including the pluggable-transport+ocm plugin. once that is done we will have additional discussion with the team to validate our own CRD+syncer are needed and if so we will create new issues to reflect the required changes.
closing for now.
